src/scss/03-widgets/_popover-odc.scss

```diff
@@ -0,0 +1,15 @@
+/* Widgets - Popover */
+////
+/// @group Widgets-Popover
+/// Widgets - Popover
+
+// IsRTL -------------------------------------------------------------------------
+///
+.is-rtl {
+	// This is a temporary fix for popover in ODC, until the platform merge the O11 popover fix into ODC
+	[data-popover] > .popover-bottom {
+		margin-left: initial;
+		margin-right: -50%;
+		transform: translateX(50%) translateY(-100%) !important; // We using the !important to override the platform inline style on popover
```

Comment:
@bmarcelino-fe why the translateY? It makes it look worse (on top) and different than when not rtl (positioned below the trigger)

Reply:
@BenOsodrac I just removed the CSS from O11 and kept the same CSS for ODC to prevent visual breaking changes. After you raised this, I've check the position of popover in IDC and contains issues on positioning based in this rule. I already reviewed the behavior and commited the new CSS only for ODC.